panther_hardware_interfaces/src/panther_system_ros_interface.cpp

```diff
@@ -28,34 +28,75 @@
 namespace panther_hardware_interfaces
 {
 
+void TriggerServiceWrapper::CallbackWrapper(
+  std_srvs::srv::Trigger::Request::ConstSharedPtr /* request */,
+  std_srvs::srv::Trigger::Response::SharedPtr response)
+{
+  try {
+    callback_();
+    response->success = true;
+  } catch (const std::exception & err) {
+    response->success = false;
+    response->message = err.what();
+
+    RCLCPP_INFO(
+      rclcpp::get_logger("PantherSystem"), "Trigger service response: %s",
+      response->message.c_str());
+  }
+}
+
+void SetBoolServiceWrapper::CallbackWrapper(
+  std_srvs::srv::SetBool::Request::ConstSharedPtr request,
+  std_srvs::srv::SetBool::Response::SharedPtr response)
+{
+  try {
+    callback_(request->data);
+    response->success = true;
+  } catch (const std::exception & err) {
+    response->success = false;
+    response->message = err.what();
+
+    RCLCPP_INFO(
+      rclcpp::get_logger("PantherSystem"), "SetBool service response: %s",
+      response->message.c_str());
+  }
+}
+
 PantherSystemRosInterface::PantherSystemRosInterface(
-  std::function<void()> clear_errors, const std::string & node_name,
-  const rclcpp::NodeOptions & node_options)
+  const std::string & node_name, const rclcpp::NodeOptions & node_options)
 {
   node_ = std::make_shared<rclcpp::Node>(node_name, node_options);
   executor_ = std::make_unique<rclcpp::executors::SingleThreadedExecutor>();
   executor_->add_node(node_);
 
   executor_thread_ = std::make_unique<std::thread>([this]() { executor_->spin(); });
 
-  clear_errors_ = clear_errors;
-
-  driver_state_publisher_ = node_->create_publisher<DriverStateMsg>(
+  driver_state_publisher_ = node_->create_publisher<panther_msgs::msg::DriverState>(
```

Comment:
```suggestion
  driver_state_publisher_ = node_->create_publisher<DriverStateMsg>(
```
Also should be chnaged in case of other services/msgs